exercism/python/sublist/sublist.py

```diff
@@ -13,11 +13,26 @@
 
 # Possible sublist categories.
 # Change the values as you see fit.
-SUBLIST = None
-SUPERLIST = None
-EQUAL = None
-UNEQUAL = None
+SUBLIST, SUPERLIST, EQUAL, UNEQUAL = range(4)
```

Comment:
This smells odd to me 🤔 
Do the tests pass?
Should `SUBLIST` be a `list` or an `int`?
What type is it according to your code?